README.md

```diff
@@ -1,36 +1,212 @@
 # heat-operator
-The Heat Operator deploys the OpenStack Heat project in a Kubernetes cluster.
+
+The Heat Operator deploys the OpenStack Heat project in a OpenShift cluster.
 
 ## Description
-This project should be used to deploy the OpenStack Heat project. It expects that there is an existing Database and Keystone service available to connect to. The config can be tuned using the `defaultConfigOverwrite` interface to override default config option in the `heat.conf` file.
 
+This project should be used to deploy the OpenStack Heat project. It expects that there is an existing Database and Keystone service available to connect to. The config can be tuned using the `defaultConfigOverwrite` interface to override default config option in the `heat.conf` file.
```

Comment:
we probably want to remove description about defaultConfigOverwrite because users can use more native interface (customServiceConfig) to customize heat.conf